ci-operator/config/openshift-priv/installer/openshift-priv-installer-master.yaml

```diff
@@ -39,6 +39,10 @@ base_images:
     name: builder
     namespace: ocp
     tag: rhel-8-golang-1.16-openshift-4.10
+  openstack-tests:
```

Comment:
Leave openshift-priv changes outside of this PR. It's going to be updated automatically via a bot after your PR merges.

https://docs.ci.openshift.org/docs/architecture/private-repositories/#openshift-priv-organization